packages/build/tests/plugins/fixtures/edge_handlers_src_dynamic/plugin.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+const { mkdir, rmdir } = require('fs')
+const { promisify } = require('util')
+
+const pMkdir = promisify(mkdir)
+const pRmdir = promisify(rmdir)
+
+const DEFAULT_EDGE_HANDLERS_SRC = 'edge-handlers'
+
+module.exports = {
+  async onPreBuild({ constants: { EDGE_HANDLERS_SRC } }) {
+    console.log(EDGE_HANDLERS_SRC === undefined)
```

Comment:
[sand] how about logging `console.log({ EDGE_HANDLERS_SRC })`.
It think it would provide more useful information in the snapshot:
```
{ EDGE_HANDLERS_SRC: undefined }
```

Reply:
`undefined` is printed differently in Node 8 :)

![log](https://user-images.githubusercontent.com/8136211/98854469-e1453d80-245a-11eb-81ea-7ac7b2441bb3.png)

This results in snapshots being different between Node.js versions.